.gitignore

```diff
@@ -14,3 +14,6 @@ logs
 
 # Dependency directory
 node_modules
+
+# SSL certificates
+.ssl
```

Comment:
not a build artifact, but required for publishing the site, so these are git-ignored